src/agent/tests/Toolbox/MetadataFactory/ChainFactoryTest.php

```diff
@@ -35,7 +35,7 @@ final class ChainFactoryTest extends TestCase
 {
     private ChainFactory $factory;
 
-    protected function setUp(): void
+    protected function setUp()
```

Comment:
please fix these, they don't need patching

```suggestion
    protected function setUp(): void
```